plugins/entando-plugin-jacms/pom.xml

```diff
@@ -189,5 +189,11 @@
                 </exclusion>
             </exclusions>
         </dependency>
+
+        <dependency>
+            <groupId>org.projectlombok</groupId>
+            <artifactId>lombok</artifactId>
+            <version>1.18.2</version>
+        </dependency>
```

Comment:
How thoroughly has this been tested? We tried adding lombok to the core at some point in the past and it broke some things so we had to role it back.

Reply:
Used lombok on the survey plugin without any issues.
Ran all the tests on the cms plugin and everything ran without any issues.